docs/getting-started.md

```diff
@@ -0,0 +1,41 @@
+In this tutorial we'll look at how to deploy the haproxy charm to provide ingress to a backend application, then configure high-avalability using the `hacluster` relation. This tutorial is done on lxd and assumes that you have a juju controller bootstrapped and a machine model to deploy charms.
+# Deploy the haproxy charm
+```
+juju deploy haproxy --channel=2.8/edge --base=ubuntu@24.04
+```
+
+After the charm has been sucessfully deployed, verify that it's serving the default page.
+```
+juju status
+# Note down the IP address of the haproxy unit
+# Save the IP address to an environment variable named HAPROXY_IP
+HAPROXY_IP=
+# Verify with curl
+curl $HAPROXY_IP
+```
+
+# Deploy the requirer application and relate to the haproxy charm
+```
+juju deploy irc-bridge
+juju relate irc-bridge haproxy
+curl $HAPROXY_IP
```

Comment:
What is expected here? Wouldnt be needed to test using the format "https://{unit_address}/{model.name}-{app_integrated.name}/"?

Reply:
I updated the docs to use the ingress URL